plugins/plugin-site/package.json

```diff
@@ -66,11 +68,13 @@
     "@jenkins-cd/gatsby-source-jenkinsplugins": "*",
     "@jenkins-cd/gatsby-transformer-rehype": "*",
     "@juggle/resize-observer": "3.4.0",
+    "@mui/material": "^6.3.1",
```

Comment:
I don't think we need a big UI package (https://packagephobia.com/result?p=%40mui%2Fmaterial) just to style a few `div`s, especially since we're overriding a lot of those styles anyway.

Reply:
Removed